### PR TITLE
Bump Ruby version from 2.7.1 to 2.7.2 and add example .env file

### PR DESCRIPTION
**Pull request by lunaisnotaboy**
Bump Ruby version from 2.7.1 to 2.7.2 and add example `.env` file with all available environment variables (taken from [`app.json`]).

[`app.json`]: https://github.com/annict/annict/blob/master/app.json

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,80 @@
+AMAZON_EN_URL=''
```
Comment (shimbaco) [inline .env.example:1]: We already have [.env.development](https://github.com/annict/annict/blob/74884acbd987f11a08c71aff97dc58dea37b9fdc/.env.development). Why does this need?

File: Gemfile [hunk 2]
```diff
@@ -94,7 +94,6 @@ group :development do
   gem "graphql-docs"
   gem "i18n-tasks"
   gem "listen" # Using with `rails s` since Rails 5
-  gem "memory_profiler"
```
Comment (shimbaco) [inline Gemfile:97]: Why the memory_profiler was removed?

**Review by shimbaco — COMMENTED** [review]
Thank you! Commented.

File: Gemfile (hunk 2 above)
Comment (lunaisnotaboy) [reply Gemfile:97]: @shimbaco It is already included on this line:
https://github.com/annict/annict/blob/b5d2741f4ebe969994ce158cf64982a3c26210a5/Gemfile#L45

Bundler will return an error if you don't remove one.

File: .env.example (hunk 1 above)
Comment (lunaisnotaboy) [reply .env.example:1]: This is for hosting Annict on somebody's own hardware.

**Review by lunaisnotaboy — COMMENTED** [review]
Hopefully cleared some things up.

File: .env.example (hunk 1 above)
Comment (shimbaco) [reply .env.example:1]: hmm, I'm not very interest that someone runs Annict on their server. If necessary, they can create it. I don't use this file, so I will probably not manage this file.

File: Gemfile (hunk 2 above)
Comment (shimbaco) [reply Gemfile:97]: oh, I didn't notice that!

**Comment by shimbaco** [comment]
Thank you for your PR! I'll merge this but I'll remove `.env.example` after that. (See https://github.com/annict/annict/pull/3086#discussion_r574235839)

**Comment by stampylongr** [comment]
@shimbaco person who runs the org here, if you want to have the `.env.example` file removed, you're gonna need to do it on the main repo, as it is currently on there.

**Pull request merged**